config/locales/models/planned_disbursement.en.yml

```diff
@@ -9,21 +9,18 @@ en:
   form:
     label:
       planned_disbursement:
-        providing_organisation_name: Providing organisation name
-        providing_organisation_type: Providing organisation type
         receiving_organisation_name: Receiving organisation name
         receiving_organisation_type: Receiving organisation type
-        providing_organisation_reference: International Aid Transparency Initiative (IATI) Reference (optional)
         receiving_organisation_reference: IATI Reference (optional)
         value: Forecasted spend amount
     legend:
       planned_disbursement:
-        providing_organisation: Providing organisation
         receiving_organisation: Receiving organisation
     hint:
       planned_disbursement:
-        providing_organisation: The organisation where this planned disbursement is coming from
-        providing_organisation_reference_html: For example, GB-GOV-13. To lookup codes or for more infomation see <a href="http://org-id.guide/" target="_blank" class="govuk-link">the organisation finder service (Opens in new window)</a>
+        receiving_organisation: Receiving organisation
+    hint:
+      planned_disbursement:
```

Comment:
This file now looks this way: 
`hint:
      planned_disbursement:
        receiving_organisation: Receiving organisation
    hint:
      planned_disbursement:
        period_end_date: End date
        period_start_date: Start date
    hint:
      planned_disbursement:`

Is this intentional? Would it be better to have all the hints grouped together?

Reply:
Good spot! Thank you!